backends/ebpf/ebpftargets.py

```diff
@@ -313,36 +393,33 @@ def create_filter(self):
         ebpfdir = os.path.dirname(__file__)
         # compiler
         args = ["gcc"]
+        args.append("-g")
         # main
         args.append("-o")
         args.append(self.tmpdir + "/filter")
         # sources
         args.append(ebpfdir + "/ebpf_runtime.c")
-        args.append(ebpfdir + "/bpfinclude/ebpf_registry.c")
-        args.append(ebpfdir + "/bpfinclude/ebpf_map.c")
+        args.append(ebpfdir + "/testinclude/ebpf_registry.c")
```

Comment:
You mean testinclude or ebpf_runtime.c?

Reply:
We should move all the hand-written C files in the runtime directory.

Reply:
It would also make sense then to move all the .cpp files into a compiler folder, or are there external dependencies on that folder structure which might break?

Reply:
The other back-ends are structured similarly, so it's better to keep the cpp files here for homogeneity.